src/main/java/stream/flarebot/flarebot/commands/moderation/NINOCommand.java

```diff
@@ -163,30 +165,18 @@ public boolean isBetaTesterCommand() {
         return true;
     }
 
-    private void listMessages(GuildWrapper wrapper, int page, TextChannel channel, User sender) {
-        EmbedBuilder eb = MessageUtils.getEmbed(sender).setTitle("**NINO Message List**");
+    private void listMessages(GuildWrapper wrapper, int page, TextChannel channel) {
         List<String> messages = wrapper.getNINO().getRemoveMessages();
-        int pageSize = 4;
-        int pages = messages.size() < pageSize ? 1 : (messages.size() / pageSize) + (messages.size() % pageSize != 0 ? 1 : 0);
-        int start;
-        int end;
-        start = pageSize * (page - 1);
-        end = Math.min(start + pageSize, messages.size());
-        if (page > pages || page < 0) {
-            MessageUtils.sendWarningMessage("That page doesn't exist. Current page count: " + pages, channel);
+        if(messages.size() == 0) {
+            MessageUtils.sendWarningMessage("No Messages for this guild!", channel);
             return;
-        } else {
-            List<String> subMessages = messages.subList(start, end);
-            if (messages.isEmpty()) {
-                MessageUtils.sendInfoMessage("There are no messages for this guild!", channel, sender);
-                return;
-            } else {
-                for (String message : subMessages) {
-                    eb.addField("Message #" + messages.indexOf(message), message, false);
-                }
-            }
         }
-        eb.setDescription("Page " + GeneralUtils.getPageOutOfTotal(page, messages, pageSize));
-        channel.sendMessage(eb.build()).queue();
+        PaginationList<String> list = new PaginationList<>(messages);
+        list.createGroups(Math.min(list.getPages(), 1));
+        PagedEmbedBuilder<String> pagedEmbedBuilder = new PagedEmbedBuilder<>(list);
+        pagedEmbedBuilder.useGroups(5);
+        pagedEmbedBuilder.setGroupPrefix("Message #");
+        pagedEmbedBuilder.setTitle("**NINO Message List**");
```

Comment:
Does this actually set the title? If so titles don't have markdown, if it doesn't should be renamed